spot-ml/src/main/scala/org/apache/spot/netflow/model/FlowSuspiciousConnectsModel.scala

```diff
@@ -87,33 +80,28 @@ class FlowSuspiciousConnectsModel(topicCount: Int,
       recordsWithIPTopicMixes.selectExpr(schema: _*).withColumnRenamed(TopicProbabilityMix, DstIpTopicMix)
     }
 
-    val scoreFunction = new FlowScoreFunction(timeCuts,
-      ibytCuts,
-      ipktCuts,
-      topicCount,
-      wordToPerTopicProbBC)
+
+    val scoreFunction = new FlowScoreFunction(topicCount, wordToPerTopicProbBC)
 
 
     val scoringUDF = udf((hour: Int,
-                          minute: Int,
-                          second: Int,
                           srcIP: String,
                           dstIP: String,
                           srcPort: Int,
                           dstPort: Int,
-                          ipkt: Long,
+                          protocol: String,
                           ibyt: Long,
```

Comment:
okay, what was inconsistent was the function signatures with the cutoffs always listed bytes first, but everywhere else packets came first and that threw me off when writing the code.... point is, good unit test coverage and a more sane design to keep these things straight could prevent wasted time